src/world/area_end/end_00/end_00_B_bowser.c

```diff
@@ -17,185 +17,191 @@ EvtScript N(EVS_JrTroopa_SpinningDodge) = {
     EVT_END
 };
 
+#if VERSION_PAL
+#define KAMMY_JRTROOPA_SPEED 1.045
+#else
+#define KAMMY_JRTROOPA_SPEED 0.87
```

Comment:
Again, we should find a value for this speed that works for both. The 0.87 is probably just insufficiently precise.